pkgs/applications/misc/mu-repo/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, buildPythonApplication, pytest, git }:
```

Comment:
I prefer to leave out stden when possible.
```suggestion
{ lib, fetchFromGitHub, buildPythonApplication, pytest, git }:
```

Reply:
Good point, fixed.